assets/app/scripts/directives/logViewer.js

```diff
@@ -269,6 +269,11 @@ angular.module('openshiftConsole')
                 streamer = null;
                 $scope.$evalAsync(function() {
                   $scope.autoScrollActive = false;
+                  // if no logs, they have already been archived.
+                  if(lastLineNumber === 0) {
+                    $scope.state = 'empty';
+                    $scope.emptyStateMessage = 'The logs are no longer available or could not be loaded.';
```

Comment:
Under what circumstances do we end up here instead of the error condition below? Do we end up here if the pod has been deleted (for instance, an old build)?